book-hu/upload_epub.py

```
@@ -0,0 +1,84 @@
+#! /usr/bin/env python3
+"""Upload EPUB to GitHub fork"""
+import json, base64, sys, requests
+sys.stdout.reconfigure(line_buffering=True)
+
+TOKEN = "ghp_jy…4OXJ"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "TOKEN"

(S105)